src/index.js

```diff
@@ -3,7 +3,8 @@ import Sinon from 'sinon';
 import ReactWrapper from './ReactWrapper';
 import ShallowWrapper from './ShallowWrapper';
 import { onPrototype } from './Utils';
-import { renderToStaticMarkup } from './react-compat';
+import reactCompat from './react-compat';
+const {renderToStaticMarkup} = reactCompat;
```

Comment:
curious, but why were these changed from imports to imports + destructure?  Is there value to doing so? is this just a style preference?